ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -243,6 +370,56 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-stack-ipi-proxy
+- as: azure-stack-upi-f28
+  cluster: build01
+  cron: 4 22 8 * *
+  steps:
+    cluster_profile: azurestack-qe
+    env:
+      E2E_RUN_TAGS: not @storage
+      EXTRACT_MANIFEST_INCLUDED: "true"
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-stack-upi
+- as: baremetal-upi-ovn-ipv4-compact-f28
+  cluster: build05
+  cron: 36 14 22 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      architecture: amd64
+      masters: "3"
+      workers: "0"
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: baremetal-lab-upi
+- as: baremetal-upi-ovn-ipv4-f28
+  cluster: build05
+  cron: 27 14 3 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      architecture: amd64
+      masters: "3"
+      workers: "2"
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: baremetal-lab-upi
+- as: baremetalds-ipi-ovn-dualstack-primaryv6-f28
```

Comment:
I don't think we need to cover all of these 3 profiles:
`baremetalds-ipi-ovn-dualstack-primaryv6-f28`
`baremetalds-ipi-ovn-ipv4-fips-f28`
`baremetalds-ipi-ovn-ipv6-fips-f28`

@lwan-wanglin @LiangquanLi930 @jianlinliu  Any suggestions on this?

Reply:
cc @sgoveas to confirm baremetal profiles.

Reply:
I think we can have 1 of each ipv4 and ipv6. Skip `baremetalds-ipi-ovn-ipv6-fips-f28`
Note that ipv6 upgrade with dev-scripts is not working at the moment (OCPQE-26572)

Reply:
keeping `baremetalds-ipi-ovn-dualstack-primaryv6-f28` for ipv6 `baremetalds-ipi-ovn-ipv4-fips-f28` for ipv4, so removed `baremetalds-ipi-ovn-ipv6-fips-f28`